### ISSUE TITLE
Inspect and improve the app accessibility

### ISSUE THREAD

**Issue by MrCsabaToth** [TrackMyIndoorWorkout/TrackMyIndoorWorkout#370]
Robert Henley & Tiphaine @ AWW October 2022
Setup Switch Access:
https://support.google.com/accessibility/android/answer/6301490
https://support.google.com/accessibility/android/answer/6122836?hl=en

**Comment by MrCsabaToth** [comment]
> Tiphaine

https://twitter.com/MobileA11y

8:16 PM | Today
Robert Henley

[web-a11y.slack.com](http://web-a11y.slack.com/)

8:18 PM | Today
Robert Henley

[moba11y.slack.com](http://moba11y.slack.com/)

8:18 PM | Today
Tiphaine

[accessiblemobileapps.slack.com](http://accessiblemobileapps.slack.com/)

8:19 PM | Today
Robert Henley

https://www.linkedin.com/in/robertallenhenley/

8:24 PM | Today
Robert Henley

Other native mobile a11y people: 
https://www.linkedin.com/in/pauljadam/

8:28 PM | Today
Robert Henley

https://www.linkedin.com/in/renatoiwashima/

8:28 PM | Today

**Comment by MrCsabaToth** [comment]
One star review by Balance, a Turkish user:
"It is not accessible to the visually impaired. However, it is not difficult to write object labels. Also, there is no Turkish language support. I don't understand how to use it. My screen readers do not read the keys. That's why it must be."

The user also needs localization: "I give 1 point for Turkish languages ​​and blinds until full support is provided." #203
